robottelo/ui/user.py

```diff
@@ -156,12 +160,14 @@ def update(self, username, new_username=None,
             default_loc=default_loc,
             select=select
         )
-        self.click(common_locators['submit'])
-
-    def admin_role_to_user(self, username):
-        """Checks if selected user has Administrator privileges otherwise
-        assign it to user.
+        if submit:
```

Comment:
not sure what you mean by that statement. We have separate test case that verify that negative scenario